types/markdown-escape/markdown-escape-tests.ts

```diff
@@ -0,0 +1,11 @@
+import * as markdownEscape from 'markdown-escape';
+import { CharacterSet } from 'markdown-escape';
```

Comment:
```suggestion
import markdownEscape = require('markdown-escape');
import { CharacterSet } from 'markdown-escape';
```
this will remove requirement to add excusion for TSLint
You would write this otherwise:
```ts
import * as markdownEscape from 'markdown-escape';
...
...
markdownEscape.CharacterSet;
```

Reply:
Resolved via c812d17557d09a0984988e6ac60b3a2b9fa63800